app/views/mailer/mfa_recommendation_announcement.text.erb

```diff
@@ -0,0 +1,24 @@
+👋 Hi <%= @user.handle %>,
+
+Today, we've announced our security-focused ambitions to the community. Attacks on the software supply chain are increasing and the second most common attack is account takeovers.
+
+On August 15, 2022, we will begin requiring maintainers of packages with more than 180 million downloads to have multi-factor authentication (MFA) enabled. Furthermore, we are officially recommending that maintainers of packages with more than 165 million downloads enable MFA. This recommendation will be repeated by the gem client. 
+
+<% if @user.mfa_disabled? %> 
```

Comment:
Friendly reminder: Plain text emails are whitespace significant, so we sadly can't indent the content in these conditional blocks.